### ISSUE TITLE
Weapon training and level requirement

### ISSUE THREAD

**Issue by Simp-N** [the-hyjal-project/bugtracker#254]
**Describe Your Issue**
On pretty much every pserver I've played on, learning new weapon skills have required level 10. On Classic there is no level requirement for this. I've spent some time trying to find info about how it actually was back in vanilla. From the vanilla period, I did find some evidence that suggests there should be no level requirement. I did find a few comments that would suggest there was. Finding evidence for lvl 10 requirements was a little easier post-vanilla. Perhaps this requirement came in TBC, and pservers emulated that. I don't know.

I have checked the [patch notes on Wowwiki](https://wowwiki.fandom.com/wiki/Patch_1.1.0) for all of vanilla and even a bit into TBC, including the so-called undocumented notes, and did not find any change about level requirements for weapon training. I've gone through the pages for weapon skills and weapon masters on Wowhead, Thottbott and such, including using wayback machine, to see if they say anything about level requirements, and to see people's comments. I tried finding old Wow videos where people went to weapon trainers. I've Googled, filtering the results the the time period. Below I will list relevant pages and comments and you can review them. User **Petage** also helped out a lot by supplying some of these links.

December 2005, [Classic Wowhead, Darnassus weapon master](https://classic.wowhead.com/npc=11866/ilyenia-moonfire#comments), comment from user
>At level 8 I checked and she only offered fist weapons at level 10. 12-29-04

January 2005, [Wowwiki page on Weapon masters](https://wowwiki.fandom.com/wiki/Weapon_master?oldid=12104)
>All weapon skills can be learned at level 1 and cost Template:S, except for Polearms which require level 20 and cost Template:G.

[Same page](https://wow.gamepedia.com/Weapon_master?oldid=12104), but from Wowpedia.

September 2005, [Wowwiki page on Class proficiencies](https://wowwiki.fandom.com/wiki/Class_proficiencies?oldid=31520)
Here's a table which shows weapon/class availability. The legend also says that it lists level requirement, and the table only mentions levels for Polearm, Dual Wield, Mail and Plate.

[Thottbot page on Crossbows](https://web.archive.org/web/20070318211344/http://www.thottbot.com/?sp=5011). Snapsnot from March 2007, but a comment from 2005
>You can train crossbows in Stormwind, and you can be any lvl. 

[Thottbot page on Guns](https://web.archive.org/web/20060428020310/http://www.thottbot.com/?sp=266). Snapshot from April 2006, but comments from 2005
>Go to Thunder Bluff and talk the to the trainer thier. Think it is level 10 

>u can learn guns if ur a warrior, rogue or hunter, and the level requirements is 1, although u need enough cash 

[Thottbot page on Two-Handed Axes](https://web.archive.org/web/20060630130433/http://thottbot.com/?sp=197). Snapshot from June 2006, comments from earlier same year:
>for shamie is there  a lvl limeted when i can learn? 

>Limit is just for guns... If you got money you can take all trainings 

Here's a clip from [Joana's leveling guide](https://youtu.be/NrSdjsrhX0s?t=1675). At 27:55 he goes to the weapon trainer in TB. There does not appear to be a level requirement written out on the panel. However, since he is above lvl 10 already, it is possible the game does not write the requirement out. This was early 2006.

April 2006, [Thottbot page for One-Handed Swords](https://web.archive.org/web/20060428013220/http://www.thottbot.com/?sp=201). It does say Level: 0, however I think the level requirements are listed a bit further down, where it says Classes: Warrior (1)...
If you look on the [page for Polearms](https://web.archive.org/web/20060630130445/http://thottbot.com/?sp=200), you'll see it says Warrior (20)... there.

**Post-vanilla**
April 2007, Here's a Wowhead [page on One-Handed Swords](https://web.archive.org/web/20070425210700/http://www.wowhead.com/?spell=201#z0z) from TBC. This one mentions a level 10 requirement.

December 2007, [Wowhead page on Two-Handed Maces](https://www.wowhead.com/spell=199/two-handed-maces#comments) has this comment on shamans. I believe this is simply refereing to the removal of the talent, rather than an actual level requirement.
>As of patch 2.3 shamans can now learn 2 handed maces and axes from level one

August 2008, [Wowhead page on One-Handed Swords](https://www.wowhead.com/spell=201/one-handed-swords#comments) has this comment:
>All skills from weapon masters are trainable at level 10 for the cost of 10 silver per skill =).

January & June 2009, [Wowhead page on Staves](https://www.wowhead.com/spell=227/staves#comments) has these comments:
>There is no level requirement for staves. (Not that i know of) 

>There is no level limit on learning Staves. The only one with level limit required is Polearms, as far as i recall

April 2009, [Forum post](https://forums.tomshardware.com/threads/shaman-and-2h-weapons.114088/)
>Moron.. Yes? Shamans can learn 2handed axe / mace from orgrimmar weapon trainer at lvl 10

Basically what I could find, during vanilla period, that suggests there was a level requirement are these three comments: 
>At level 8 I checked and she only offered fist weapons at level 10. 12-29-04

>Go to Thunder Bluff and talk the to the trainer thier. Think it is level 10 

>Limit is just for guns... If you got money you can take all trainings 

and I aint even sure if the two latter ones should even count. Meanwhile I think some of the evidence against level requirement are a bit more substantial, with the wiki articles, database pages, the video. However, that it's always been a level requirement on pservers is also an argument, or at least you'd want to be sure before you change it. What do you think about this?

**Comment by Simp-N** [comment]
If someone has some relevant link or insight, I'd love to know

**Comment by Nolin7777** [comment]
Thanks for the research you have done already.
It looks like some weapon skills does have a requirement and some haven't.
We need to find out what's the case for every single skill.
